pages/common/dotnet-build.md

```diff
@@ -0,0 +1,32 @@
+# dotnet build
+
+> Builds a .NET application and its dependencies.
+> More information: <https://docs.microsoft.com/dotnet/core/tools/dotnet-build>.
+
+- Compile the project or solution in the local path:
```

Comment:
Do you mean the current directory here, or the `PATH` environment variable?
```suggestion
- Compile the project or solution in the current directory:
```
If the former, then this is more consistent with other pages :-)